core/include/moveit/task_constructor/stages/connect.h

```diff
@@ -73,7 +73,13 @@ class Connect : public Connecting
 		WAYPOINTS = 1
 	};
 
-	using GroupPlannerVector = std::vector<std::pair<std::string, solvers::PlannerInterfacePtr> >;
+	struct PlannerIdTrajectoryPair
```

Comment:
Thanks for doing this!

My comment was less about the struct vs. pair data structure, and the fact that if you ever had to add more data to this it would be challenging. Same still holds here because the struct has `Pair` in its name. Is there some more generic name we can use like `TrajectoryInfo`?

... and same with the variable names that use this data type, which are still called `vector` and `pair`

Reply:
I think we should keep it like this for now because the name describes what it is and if we add more to this data structure in the future the name can be updated accordingly. Despite the fact that this could also be of datatype pair, I think making it a struct made the code more readable :+1:

Reply:
`TrajectoryInfo` is very generic and MTC has already a datatype called like this.

Reply:
Sounds good -- I can't think of anything better anyhow.